app/_layouts/extension.html

```diff
@@ -178,21 +239,37 @@
   {% if config_required_fields_yaml == 'config: ' %}config:
     <optional_parameter>: <value>{% else %}{{ config_required_fields_yaml }}{% endif %}
 ```
-{% endnavtab %}
-{% endunless %}
-{% endnavtabs %}
-{% endif %}
 
-`<service>` is the `id` or `name` of the Service that this plugin
+`<service>` is the `id` or `name` of the service that this plugin
   configuration will target.
 
-{% if page.params.api_id %}
-  > **Note:** The legacy API entity has been **deprecated** in favor of Services
-  since <a href="https://github.com/Kong/kong/blob/master/CHANGELOG.md#0130---20180322">
-  CE 0.13.0</a> and
-  <a href="https://docs.konghq.com/enterprise/changelog/#0-32">EE 0.32</a>.
-{% endif %}
+{% endnavtab %}
+{% endunless %}
+{% unless page.params.konnect_examples == false %}
+{% navtab Konnect %}
```

Comment:
Should probably have "Konnect SaaS" as the tab title, even though it'll stretch the tab. 
Maybe shorten the "Kong Manager" tab title to just "Manager" to make up for it.

Reply:
yep, will fix